src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,30 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("completeOrder method was called. Input param: userId = {}", userId);
         List<Product> products = getAllProductsFromShoppingCart(userId);
```

Comment:
The `getAllProductsFromShoppingCart` method currently returns a hardcoded list of products. Consider replacing this with actual logic to retrieve products from a shopping cart associated with the user when moving to production.

Reply:
it was not part of the task